bsc.typechecker/generics/Makefile

```diff
@@ -0,0 +1,5 @@
+# for "make clean" to work everywhere
+
+CONFDIR = $(realpath ..)
```

Comment:
When you moved this to a lower directory, you needed to add `/..` to the path here -- otherwise `make check` in this directory doesn't work

Reply:
Fixed.